src/Data/Message/MessageEditingConfig.ts

```diff
@@ -5,7 +5,5 @@ provideEditingConfig(Message, {
   title: 'Message',
   attributes: {
     attachments: { title: 'Attachments' },
```

Comment:
* [ ] Q: What happened to `subjectId`?

Reply:
How to define a reference where we don't know the `to`?
It could be `Ticket`, `Quote`, `Order`, ...

Reply:
As far as I know there are unspecific future plans to introduce linking to multiple class names. For now I'll reintroduce `subjectId` to the editing config.